components/builder-web/webpack.conf.js

```diff
@@ -1,55 +1,49 @@
 'use strict';
 
 const webpack = require('webpack');
+const path = require('path');
 const isProduction = process.env.NODE_ENV === 'production';
 
-let loaders = [
-    { test: /\.ts$/, loader: 'awesome-typescript-loader' },
-    { test: /\.html$/, loader: 'raw-loader' }
+let rules = [
+    { test: /\.ts$/, use: [{ loader: 'awesome-typescript-loader' }] },
+    { test: /\.html$/, use: [{ loader: 'raw-loader' }] },
+    {
+        test: /\.ts$/,
+        enforce: 'pre',
+        use: [{
+            loader: 'tslint-loader',
+            options: {
+                emitErrors: true,
+                failOnHint: true
+            }
+        }]
+    }
 ];
 
 let plugins = [];
 let devtool = 'source-map';
 
 if (isProduction) {
     devtool = false;
-
-    // Set up compression to only happen if NODE_ENV = production
-    loaders.push({ test: 'app.js', loader: 'uglify' });
-    plugins.push(
-        new webpack.optimize.UglifyJsPlugin({
-            compress: {
-                drop_debugger: false,
-                warnings: false
-            },
-            mangle: false,
-            sourceMap: false
-        })
-    );
```

Comment:
This is now default behavior: https://webpack.js.org/configuration/optimization/#optimizationminimize